src/rf/js/src/home/components/map.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+var React = require('react');
+
+var Map = React.createBackboneClass({
+    render: function() {
+        return (
+            <div className="map-container">
+                <div id="map"></div>
+            </div>
+        );
+    }
+});
```

Comment:
There may be a way to coalesce this with the `MapView` but I haven't yet figured out a neat way of doing it while also being able to incorporate this within other React components.